packages/block-editor/src/hooks/fit-text.js

```diff
@@ -94,8 +104,13 @@ function useFitText( { fitText, name, clientId } ) {
 			styleElement.textContent = css;
 		};
 
-		optimizeFitText( blockElement, blockSelector, applyStylesFn );
-	}, [ blockElement, clientId, hasFitTextSupport, fitText ] );
+		// Avoid very jarring resizes when a user is actively editing the
+		// block. Placing a ceiling on how much the block can grow curbs the
+		// effect of the first few keypresses.
+		const maxSize = isSelectedRef.current ? 200 : 600;
```

Comment:
What do you think about 

```suggestion
		const maxSize = isSelectedRef.current ? 200 : undefined;
```